translator/reader/registry.cpp

```diff
@@ -114,6 +114,7 @@ void UsdArnoldReaderRegistry::registerReader(const std::string &primName, UsdArn
     _readersMap[primName] = primReader;
 }
 
+#ifdef ARNOLD_USE_VIEWPORT_API
```

Comment:
Why introduce a new `ARNOLD_USE_VIEWPORT_API` macro when we could just check the Arnold version is above 6?

`#if AI_VERSION_ARCH_NUM >= 6`

Reply:
Introducing a named macro for each feature we want to enable or disable based on a dependency's version makes the code more readable, and it's easier to see which segment belongs to which feature.

Reply:
I replaced the macro `ARNOLD_USE_VIEWPORT_API` by a test on `AI_VERSION_ARCH_NUM` as proposed. Added the comments on why we do so